internal/command/init.go

```diff
@@ -715,8 +715,8 @@ func (c *InitCommand) getProviders(ctx context.Context, config *configs.Config,
 			default:
 				diags = diags.Append(tfdiags.Sourceless(
 					tfdiags.Error,
-					"Failed to query available provider packages",
-					fmt.Sprintf("Could not retrieve the list of available versions for provider %s: %s",
+					"Failed to resolve provider packages",
+					fmt.Sprintf("Could resolve provider %s: %s",
```

Comment:
I think you lost the negation here

Reply:
Thanks, fixed in [f0a15fc](https://github.com/opentofu/opentofu/pull/1003/commits/f0a15fc0339a44f84a0344b1120dfbbd93df5f00)